app/assets/stylesheets/govuk_publishing_components/components/_step-by-step-nav.scss

```diff
@@ -237,7 +237,9 @@ $top-border: solid 2px $grey-3;
   }
 
   .gem-c-step-nav__button {
-    @include govuk-focusable;
+    &:focus {
+      outline: $govuk-focus-width solid $govuk-focus-colour;
+    }
```

Comment:
Not sure what's happened here but the show/hide all button on step navs (which has this class) seems to have lost its focus state.

<img width="802" alt="Screen Shot 2019-08-05 at 17 29 14" src="https://user-images.githubusercontent.com/861310/62480028-af98ed80-b7a6-11e9-9888-bc00b03861f7.png">

Reply:
Perhaps one for a follow up pull request but that style will not pass non-text guidelines.

https://design-system.service.gov.uk/get-started/focus-states/

Reply:
I did an update on the step-by-step-nav to improve focus states.

<img width="960" alt="Screen Shot 2019-08-07 at 11 08 42" src="https://user-images.githubusercontent.com/788096/62614689-cef95d00-b903-11e9-9caa-002b2fc45536.png">

This component is also a good candidate for an overhaul looking at the use of `!important`

Reply:
Looking good, however there looks like some issues around hover + focus:

<img width="166" alt="Screen Shot 2019-08-07 at 13 46 59" src="https://user-images.githubusercontent.com/2445413/62623959-f1967080-b919-11e9-8613-cc5f083dff8f.png">

Reply:
Yep, I'm thinking how to tackle that by untangling styles rather than chaining selectors.